extensions/amp-iframe/1.0/README.md

```diff
@@ -0,0 +1,197 @@
+# Bento Iframe
+
+## Usage
+
+Displays an iframe.
+
+Use Bento Iframe as a web component `<bento-iframe>`, or as a Preact/React functional component `<BentoIframe>`.
+
+### Web Component
+
+You must include each Bento component's required CSS library to guarantee proper loading and before adding custom styles. Or use the light-weight pre-upgrade styles available inline. See [Layout and style](#layout-and-style).
+
+The examples below demonstrate use of the `<bento-iframe>` web component.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-iframe
+```
+
+```javascript
+import '@ampproject/bento-iframe';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
```

Comment:
This can be removed, we will be providing this as an optional tag under a generic "Browser support" section.
```suggestion
```